doc/modules/clustering.rst

```diff
@@ -1250,6 +1250,9 @@ clusters (labels) and the samples are mapped to the global label of the nearest
   - If the number of instances of data needs to be reduced, or if one wants a
     large number of subclusters either as a preprocessing step or otherwise,
     BIRCH is more useful than MiniBatchKMeans.
+  - See :ref:`sphx_glr_auto_examples_cluster_plot_birch_vs_minibatchkmeans.py` as an example
```

Comment:
For consistency, could you make a new rubric "Examples" at the end of the section "2.3.10. BIRCH" and place the link there? You can compare the section "2.3.9. OPTICS" for how to do this. Also for consistency, could you remove the "See" and replace it with a star (compare other links)? This will start a list. I would also shorten the description a bit by removing the part "applied to a synthetic dataset ..." to keep the main focus on the computation time.

Reply:
Thank you @marenwestermann I've done the suggested improvements. You can check if they are suitable!!!

Reply:
Thank you! For consistency with the other example descriptions: could you add a colon behind the link and write something like, "A comparison of computational run times between the two estimators," ? (Note that the word "method" is reserved for the programming concept). I didn't see this earlier, apologies.

Reply:
No problem @marenwestermann! The update is ready for your review, thanks!